src/_cffi_src/openssl/objects.py

```diff
@@ -9,6 +9,14 @@
 """
 
 TYPES = """
+typedef struct {
+    int type;
+    int alias;
+    const char *name;
+    const char *data;
+} OBJ_NAME;
```

Comment:
This struct isn't opaque in 1.1.0 so we can bind it this way -- how is it used in CPython right now though?